src/app/components/lobby/dialogs/open-game-popup/open-game-popup.component.ts

```diff
@@ -67,37 +74,40 @@ export class OpenGamePopupComponent implements OnInit {
   selectedDeck = 0;
 
   selectedDeckExt: TileSet = undefined;
-  tileListAsSet: Map<number, {bt: BoardTile, count: number}> = new Map<number, {bt: BoardTile; count: number}>();
+  tileListAsSet: Map<number, { bt: BoardTile; count: number }> = new Map<number, { bt: BoardTile; count: number }>();
   popoverVisible = false;
 
-  @ViewChild('deckCarousel', {static : true}) deckCarousel: NgbCarousel;
-
+  @ViewChild('deckCarousel', { static: true }) deckCarousel: NgbCarousel;
 
   // Onclick FieldList
   //  multiples as x2 - tag
   //  only list of pictures
```

Comment:
please remove useless comments. Or if they are needed please elaborate what they seem to be saying as I do not understand this one.